src/marketplace/marketplace.page.jsx

```diff
@@ -49,21 +58,27 @@ const MarketplacePage = () => {
             {isLoading ? (
                 <p>Loading</p>
             ) : (
-                <div className="float-right">
+                <div className="mt-6 grid grid-cols-1 gap-y-10 gap-x-6 sm:grid-cols-2 lg:grid-cols-4 xl:gap-x-8">
                     {nfts.map((nft) => (
-                        <div key={nft.id} className="w-1/4 inline-block">
-                            Symbol: {nft.symbol} <br />
-                            Name: {nft.name} <br />
-                            Address: {nft.address}
-                            <br />
-                            Creator: {nft.creator.id}
-                            <br />
-                            Owner: {nft.owner.id}
-                            <br />
-                            ProviderURL: {nft.providerUrl}
-                            <br />
-                            <br />
-                            <br />
+
+                        <div id={nft.id} class="card text-black block p-6 rounded-lg shadow-lg bg-white max-w-sm">

```

Comment:
`class` isn't used in React
Like elsewhere use `className`
`card` isn't defined anywhere use Tailwind classes only